handbook/engineering/releases/release_issue_template.md

```diff
@@ -31,12 +30,17 @@ Arguments:
   yarn run release changelog:cut $MAJOR.$MINOR.0
   ```
 - [ ] Create the `$MAJOR.$MINOR` branch off the CHANGELOG commit in the previous step: `git branch $MAJOR.$MINOR && git push origin $MAJOR.$MINOR`.
+
+## Release candidates
```

Comment:
```suggestion
```

As it stands this makes me think I might do this on the 20th instead of immediately after branch cut, so I suggest removing this header.